docs/user-guide/rules/performance-budget.md

```diff
@@ -0,0 +1,147 @@
+       (number of redirects * RTT) +
+       (total number of requests * TCP slow-start phase) +
+       (total size of resources / bandwidth)
+```
+
+This is the list of things taken into account:
+
+* Everything is a first load, no values are cached, and no connections are
+  opened.
+* [`RTT` (Round-Trip Time)][rtt] is fixed and changes depending on the
+  configured network (more about this in #). It assumes all servers respond
+  instantly and in the same amount of time.
+* [`DNS lookup`][dns lookup]: Every hostname resolution requires 1 RTT,
+  imposing latency on the request and blocking the request while the lookup is
+  in progress.
+* [`TCP handshake`][three-way handshake]: Each request requires a new TCP
+  connection. TCP connections require 1 RTT before start sending information to
+  the server. There's no connection reuse and the maximum number of connections
+  to a domain (usually 6) is ignored.
+* [`TCP slow-start phase`][slow-start phase]: The values used to calculate the
+  duration are:
+  * `cwnd`: 10 network segments
+  * `rwnd`: 65,535 bytes (no `TCP window scaling`)
+  * `segment size`: 1460 bytes
+  After this phase, the full bandwidth of the connection is used to download
+  the remaining.
+* [`TLS handshake`][tls handshake]: New TLS connections usually require two
+  roundtrips for a "full handshake". However, there are ways of requiring only
+  1 RTT like [`TLS False Start`][tls false start] and [`TLS Session
+  Resumption`][tls session resumption]. This rule assumes the optimistic
+  scenario.
+* Redirects: We simplify the redirects and assume the connection is reused and
+  to the same server. If the redirect is to another domain, the penalty will be
+  even greater.
+* The server doesn't use [`HTTP2`][http2].
+
+## Can the rule be configured?
+
+You can change the type of connection and/or the target load time as follows:
+
+```json
+{
+    "performance-budget": ["error", {
+        "connectionType": "Dial",
+        "loadTime": 10
+    }]
+}
+```
+
+The possible values and the associated speeds for `connectionType` are:
+
+| Value  |       In |      Out |   RTT |
+| -------|----------|----------|-------|
+| FIOS   |  20 Mbps |   5 Mbps |   4ms |
+| LTE    |  12 Mbps |  12 Mbps |  70ms |
+| 4G     |   9 Mbps |   9 Mbps | 170ms |
+| Cable  |   5 Mbps |   1 Mbps |  28ms |
+| 3G     | 1.6 Mbps | 768 Kbps | 300ms |
+| 3GFast | 1.6 Mbps | 768 Kbps | 150ms |
+| DSL    | 1.5 Mbps | 384 Kbps |  50ms |
+| 3GSlow | 400 Kbps | 400 Kbps | 400ms |
+| 3G_EM  | 400 Kbps | 400 Kbps | 400ms |
+| 2G     | 280 Kbps | 256 Kbps | 800ms |
+| Edge   | 240 Kbps | 200 Kbps | 840ms |
+| Dial   |  49 Kbps |  30 Kbps | 120ms |
+
+`loadTime` has to be a number greater than `1` and indicates the time in
+seconds to load all the resources.
+
+### Examples that **trigger** the rule
+
+Assuming a `3G Fast` connection, the following will fail:
+
+* A site with 2 redirects and 2 images over 362KB each
```

Comment:
I think these examples are a bit confusing. Maybe also specify the HTML and it's size?